ArduCopter/Copter.cpp

```diff
@@ -92,6 +92,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK(rc_loop,              100,    130),
     SCHED_TASK(throttle_loop,         50,     75),
     SCHED_TASK(update_GPS,            50,    200),
+    SCHED_TASK_CLASS(GCS,                  (GCS*)&copter._gcs,          send_planck_stateinfo,    100, 100),
```

Comment:
Didn't state info used to be sent at 50Hz? (I mean higher is better but is it ok to increase it?)

Reply:
The task runs at 100hz, but the states are generated at whatever rate they were requested (the streamrate).  `planck_ctrl` requests messages at 50hz.  The reason this is at 100hz is so that, if there was a slow down, we don't wait a full 20ms to send a "late" state message (I encountered this on the bench).

Reply:
got it 👍